doc/man7/provider.pod

```diff
@@ -218,7 +218,7 @@ L<EVP_CipherInit_ex(3)>, the actual implementation to be used is
 fetched implicitly using default search criteria.
 
 Implicit fetching can also occur with functions such as
-L<EVP_PKEY_CTX_derive_init_ex(3)> where a NULL algorithm parameter is
+EVP_PKEY_CTX_derive_init_ex() where a NULL algorithm parameter is
```

Comment:
This is a typo, the correct name is `EVP_PKEY_derive_init_ex`

Reply:
fixed, thanks!